### PR TITLE
Fix for change that broke some keyvault commands on 0.9.3

### PR DESCRIPTION
**Pull request by msfcolombo**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@msfcolombo__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (fcolombo). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: lib/util/utils.js [hunk 1]
```diff
@@ -205,7 +205,8 @@ exports.createEventsClient = function (subscription) {
 };
 
 exports.createKeyVaultClient = function (subscription, vaultUri) {
-  var factoryMethod = require('azure-keyvault').createKeyVaultClient;
+  var keyvault = require('azure-keyvault');
+  var factoryMethod = keyvault.createKeyVaultClient;
```
Comment (yugangw-msft) [inline lib/util/utils.js]: The change doesn't appear make any difference from the current code one. Am i missing anything? What was the error log

Comment (msfcolombo) [inline lib/util/utils.js]: The change reintroduces the keyvault variable, which is used below on line 211 (not showing on the diff). The error log is an exception about the undefined variable keyvault.

Comment (yugangw-msft) [inline lib/util/utils.js]: it makes sense now.

**Pull request merged**